exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/statements/BatchUpsertStatement.kt

```diff
@@ -13,7 +11,7 @@ import org.jetbrains.exposed.sql.vendors.currentDialect
  * @param table Table to either insert values into or update values from.
  * @param keys (optional) Columns to include in the condition that determines a unique constraint match. If no columns are provided,
  * primary keys will be used. If the table does not have any primary keys, the first unique index will be attempted.
- * @param onUpdate List of pairs of specific columns to update and the expressions to update them with.
+ * @param onUpdate Lambda accepting a list of pairs of specific columns to update and the expressions to update them with.
```

Comment:
I'll change this text to reflect the proper type when I address any other changes.